sdk/go/common/workspace/project.go

```diff
@@ -104,13 +104,29 @@ type ProjectConfigItemsType struct {
 }
 
 type ProjectConfigType struct {
```

Comment:
Wonder if its worth splitting this type DU style and writing custom marshallers. Maybe some thing to play with when we're not under time pressure for the release this week.

Reply:
I suggest we keep this in mind for later, it would require a refactor everywhere but it would make the code somewhat nicer when we model the fact that value is a special thing